src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts

```diff
@@ -54,15 +54,15 @@ export abstract class SimpleFindWidget extends Widget {
 	private _foundMatch: boolean = false;
 
 	constructor(
-		@IContextViewService private readonly _contextViewService: IContextViewService,
-		@IContextKeyService contextKeyService: IContextKeyService,
-		private readonly _state: FindReplaceState = new FindReplaceState(),
-		private readonly _options: IFindOptions,
-		private readonly _keybindingService?: IKeybindingService
+		readonly state: FindReplaceState = new FindReplaceState(),
```

Comment:
Oh this was private before too, need to remove readonly?

Reply:
yeah I wasn't sure if readonly was used to make it obvious that isn't changed/set

Reply:
I've been bitten by doing this exact thing before, don't think it's possible to declare a parameter variable as readonly unless it's a member

Reply:
options were also readonly before